### ISSUE TITLE
documentation discrepancy for dfa.minify() 

### ISSUE THREAD

**Issue by colette-b** [caleb531/automata#59]
Hello,
The documentation says "retain_names=False" is default,
but in the code, it's actually "retain_names=True".

**Comment by caleb531** [comment]
Oh dear, thank you for the catch, @colette-b! What do you think the default should be?

I am leaning towards `retain_names=False` since that is what's already in the documentation. Plus, most of the other DFA operation methods already have `retain_names=False` as the respective default for those methods.

cc @Tagl (who originally implemented this)

**Comment by colette-b** [comment]
I agree, `retain_names=False`  feels like the more natural default :)

**Comment by Tagl** [comment]
I don't have a strong opinion on which default is more natural. I do however think it is much more common to not care about the names so False seems like a fine choice.

**Comment by caleb531** [comment]
Closing this ticket as the documentation has now been corrected with the recent v7 release.
